include/client/libcurve.h

```diff
@@ -382,10 +401,16 @@ inline bool operator==(const UserInfo& lhs, const UserInfo& rhs) {
 
 struct OpenFlags {
     bool exclusive;
-
     OpenFlags() : exclusive(true) {}
 };
 
+struct OpenFileContext_t {
```

Comment:
I think you can put the definition into client_common.h, and what's the meaning of suffix `_t`?